phpseclib/Crypt/Rijndael.php

```diff
@@ -190,29 +190,6 @@ class Rijndael extends Base
     var $kl;
 
     /**
-     * Default Constructor.
```

Comment:
I think we may have a misunderstanding here. I was not trying to say you're wrong or anything similar. I was surprised to see this change actually made it into the repository. I agree it was likely caused by a merge. I'd be interested in knowing which merge. Specifically, whether I am to blame for this. :-P